src/pages/home/report/ReportActionItemSingle.tsx

```diff
@@ -85,7 +85,9 @@ function ReportActionItemSingle({
     const personalDetails = usePersonalDetails() ?? CONST.EMPTY_OBJECT;
     const policy = usePolicy(report?.policyID);
     const delegatePersonalDetails = personalDetails[action?.delegateAccountID ?? ''];
-    const actorAccountID = ReportUtils.getReportActionActorAccountID(action, iouReport);
+    const ownerAccountID = iouReport?.ownerAccountID ?? action?.childOwnerAccountID;
+    const isReportPreviewAction = action?.actionName === CONST.REPORT.ACTIONS.TYPE.REPORT_PREVIEW;
+    const actorAccountID = isReportPreviewAction && ReportUtils.isPolicyExpenseChat(report) ? ownerAccountID : ReportUtils.getReportActionActorAccountID(action, iouReport);
```

Comment:
Move the new logic to `ReportUtils.getReportActionActorAccountID`

Reply:
Updated